provider/pkg/gen/dotnet-templates/helm/ChartBase.cs

```diff
@@ -110,13 +110,13 @@ protected ChartBase(string releaseName, Union<ChartArgs, LocalChartArgs> args, C
                     // > looked up or retrieved in-cluster will be faked locally. Additionally, none
                     // > of the server-side testing of chart validity (e.g. whether an API is supported)
                     // > is done.
-                    var flags = new List<string>(new[]
+                    var flags = new List<string>
```

Comment:
No need to create an intermediate array to pass to the list -- we can pass the initial items directly using the collection initializer syntax.